lumenes_api/lumenes/views.py

```diff
@@ -1,10 +1,20 @@
 from django.shortcuts import render
 from django.contrib.auth.models import User, Group
-from rest_framework import viewsets, permissions
-from lumenes_api.lumenes.serializers import UserSerializer
+from rest_framework import viewsets, permissions, generics
+
+from lumenes_api.lumenes.models import Video
+from lumenes_api.lumenes.serializers import UserSerializer, VideoSerializer
 
 
 class UserViewSet(viewsets.ModelViewSet):
     permission_classes = (permissions.IsAdminUser,)
     queryset = User.objects.all().order_by('-date_joined')
     serializer_class = UserSerializer
+
+
+class ListVideosView(generics.ListAPIView):
```

Comment:
Creo que es mejor usar viewsets en vez de un ListAPIView porque asi puedes hacer muchas mas cosas, intenta adaptarlo de esta manera ya que he planteado la api de esta forma

https://www.django-rest-framework.org/api-guide/viewsets/

https://stackoverflow.com/questions/41379654/difference-between-apiview-class-and-viewsets-class#:~:text=APIView%20allow%20us%20to%20define,%2C%20RETRIEVE%2C%20UPDATE%2C%20etc.&text=The%20functions%20you%20add%20to,add%20to%20the%20ViewSet%20class.